python/src/aiconfig/editor/client/src/Editor.tsx

```diff
@@ -276,6 +291,9 @@ export default function Editor() {
               color: "white",
               borderRadius: "0",
               height: "auto",
+              "&:hover": {
+                background: "#ff46f8",
```

Comment:
One thing to look into afterwards -- we can set a [primaryColory](https://mantine.dev/theming/colors/) with the 10 different shades and it will automatically apply to all button backgrounds, hover, highlight around text area, etc. without needing to manually do it.